crates/solidity/testing/snapshots/cst_output/InterfaceDefinition/sample_counter/generated/0.4.11.snap

```diff
@@ -1,6 +1,44 @@
 # This file is generated automatically by infrastructure scripts. Please don't edit by hand.
 
-Errors: 0
+Error: subtrahend matches minuend
+   ╭─[crates/solidity/testing/snapshots/cst_output/InterfaceDefinition/sample_counter/input.sol:3:22]
+   │
+ 3 │     function count() external view returns (uint);
+   ·                      │ 
+   ·                      ╰─ Error occurred here.
+───╯
+
+Error: subtrahend matches minuend
```

Comment:
Per the PR title, this is fixing a few bugs. I suggest including some test cases for the new/fixed inputs, as part of this PR.

For example (for #256), we can generate the test cases (and their snapshots) by running the following command:

```sh
mkdir -p "crates/solidity/testing/snapshots/cst_output/Keyword/returns_keyword"
echo "returns" > "crates/solidity/testing/snapshots/cst_output/Keyword/returns_keyword/input.sol"
"./scripts/cargo/test.sh" # to generate the output/snapshots
```

Reply:
I agree, as much as I would rather not.